deploy/helm/scf/assets/operations/instance_groups/api.yaml

```diff
@@ -170,12 +170,56 @@
       internal: 9024
     run:
       healthcheck:
+        ccng_monit_http_healthcheck:
+          readiness:
+            # This job exists just to healthcheck cloud_controller_ng; we're
+            # already doing that separately.
+            exec:
+              command: [/bin/true]
+          liveness:
+            exec:
+              command:
+              - /usr/bin/pgrep
+              - --full
+              - --exact
+              - bash /var/vcap/jobs/cloud_controller_ng/bin/ccng_monit_http_healthcheck
         cloud_controller_ng:
           readiness: &cloud_controller_ng_readiness
+            httpGet:
+              path: /healthz
+              port: 9022
```

Comment:
This is _technically_ not correct; it's actually checking at the endpoint behind nginx.  However, the version of curl in our stemcell is too old to support talking to unix sockets.  We should be able to move to `curl --unix-socket` once we bump our stemcell to Leap/SLE 15.

Reply:
I actually tried that first; it didn't work for some reason (I kept getting a `bad request` response, which would have failed to healthcheck).  I can try it again (I don't have a v3 deployment ready at the moment).

Reply:
If you "Allow edits from maintainers", I can take this over and push commits to your fork, so I don't interrupt your current work.

Reply:
Fixed in 215d8e8101fec5f68087915249cd475381a6289e — I needed to add `-q 2` to the `nc` command line to keep the connection open for a bit, in order for `nc` to read the response back.

(_Allow edits from maintainers_ was on when the PR was created)